src/pages/home/Home.js

```diff
@@ -0,0 +1,79 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './Home.css';
+import {Link} from "react-router-dom";
+import Header from "../../components/header/Header";
+import redditLogo from '../../assets/logo.png';
+import {ReactComponent as LoaderImage} from "../../assets/reddit-app-loader.svg";
+import numberWithDots from "../../helpers/DotNotation";
+import TruncateString from "../../helpers/TruncateText";
+import Footer from "../../components/footer/Footer";
+
+function Home() {
+    const [posts, getPosts] = useState(null);
```

Comment:
```suggestion
    const [posts, setPosts] = useState(null);
```
Omdat als je het `getPosts` noemt, dan verwacht ik iets terug te krijgen uit die functie in plaats van dat ik iets set